neo/test/iotest/test_cedio.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+
+from neo.io import CedIO
+from neo.test.iotest.common_io_test import BaseTestIO
+
+
+class TestCedIO(BaseTestIO, unittest.TestCase, ):
+    ioclass = CedIO
+    entities_to_test = [
+        'spike2/m365_1sec.smrx',
+        'spike2/File_spike2_1.smr',
+        'spike2/Two-mice-bigfile-test000.smr'
+    ]
+    entities_to_download = [
+        'spike2'
```

Comment:
@samuelgarcia so you don't think the `entities_to_download` should be common to the rawio and io implementations?

Reply:
This could be but making hineritence between class for testing is painfull.
And it is outside the PR I think.

Reply:
I remember you did something like this for other IOs
```suggestion
        TestCedRawIO.entities_to_download
```

Reply:
For entities to test it makes sens. For the downloading, now it is only one str because it is folder based now with datalad.
But we could do (outside this PR of course...)